app/uk/gov/hmrc/hmrcemailrenderer/templates/passcode/agentMappingInvitation.scala.html

```diff
@@ -0,0 +1,9 @@
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Thanks for agreeing to help develop Making Tax Digital") {
+<p style="margin: 0 0 30px; font-size: 19px;">Dear @{params("name")}</p>
+<p style="margin: 0 0 30px; font-size: 19px;">Thanks for agreeing to help develop Making Tax Digital.</p>
+<p style="margin: 0 0 30px; font-size: 19px;">This email contains a link to the prototype you’ve agreed to test: adding existing Self Assessment clients to an Agent Services account.</p>
```

Comment:
Could you please replace all the `’` with `'` in the templates? We tend to use standard single/double quotes.